CONTRIBUTING.md

```diff
@@ -68,8 +68,10 @@ please consider submitting another pull request with the corresponding [document
 
 ### Setting up the Build System
 
-The Leaflet build system uses [Node](http://nodejs.org/), and the [Jake](http://jakejs.com/) Javascript build tool.
-To set up the Leaflet build system, install Node then run the following commands in the project root to install dependencies:
+The Leaflet build system uses [Node](http://nodejs.org/).
+To set up the Leaflet build system, install [NodeJS](https://nodejs.org/en/).
+Note, that on Ubuntu/Debian you may need to [create a `node` symlink for nodejs package](https://stackoverflow.com/a/18130296).
+Then run the following commands in the project root to install dependencies:
```

Comment:
Let's remove the Ubuntu/Debian mention and instead make sure you never need to call `node` directly, only through `npm run`.

Reply:
I do not call `node` directly, but there is [shebang line on the beginning of every script](https://github.com/Leaflet/Leaflet/pull/5828/files#diff-737e967a491eb091ac9d9d19c9078b0eR1), that tells the system what interpreter to pass that file to (only on Unix-like platforms, Windows will just ignore that line and use extension to determine what executable will interpret it).

Note, that Jake also resolves `node` executable that way - see [Jake's CLI](https://github.com/jakejs/jake/blob/master/bin/cli.js#L1).

Reply:
As far as I understand, the line is only necessary if you call the script directly, e.g. `./script.js` from the command line, which is not the way we suggest to run the scripts (since they're part of `npm` scripts). So removing the note should keep the guide clearer without consequences.

Reply:
Actually, it's necessary even when calling script through NPM.

For example, you may want to run a bash script through NPM - and it shouldn't be executed by `node`. That's why `#!/usr/bin/env node` is necessary.

There is a mention about that in [NPM docs](https://docs.npmjs.com/files/package.json#bin) (bottom of `bin` section).
[More info about that shebang line](http://www.nodejsconnect.com/blog/articles/command-line-node).

Reply:
The only Bash script we have in the Leaflet repo is `publish.sh`, and because it's Bash, it's irrelevant to the note about `nodejs` executable on Debian.

Reply:
Bash script was just an example :)
I just wanted to clear that shebang line is necessary in order to tell NPM which executable should be used for particular script.
I added Ubuntu/Debian mention to docs only because of `nodejs`/`node` packages [naming conflict on those platforms](https://stackoverflow.com/a/18130296)

Reply:
Hi,

Trying to get this PR going through: if I understand correctly @mourner's comments:
> remove the Ubuntu/Debian mention

…and…

> keep the guide clearer

…the idea would be to have a simple guide, even if we have to let some specificities (like properly linking to node in this case) to the community to figure out (there is plenty resources on how to install node). IIRC, the standard install on Ubuntu installs `node` in the path, so this manual step would not be the common case. Hence we could reasonably keep it out indeed.

As for the shebang line added by @cherniavskii, it is necessary in order to have the file executed by `node` without having to call `node` explicitly in the CLI. However, as noted by both @mourner and @cherniavskii, since all these scripts are actually called as `npm` package scripts, we can simply include the `node` call in those scripts, and avoid having this shebang line cause [other troubles](http://www.nodejsconnect.com/blog/articles/command-line-node#shebang-platform-issues).

E.g.:
package.json
```json
"docs": "node ./build/docs.js"
```

What do you think?

Reply:
@ghybs I didn't know about those troubles using shebang line.
Indeed, calling node in npm scripts sounds reasonable.

>the standard install on Ubuntu installs node in the path

Yes, I confirm. I have installed node on Ubuntu lately and `node` alias works just fine.

I'll remove shebang lines and update docs.
Thanks!

Reply:
@ghybs done!

Reply:
@cherniavskii thank you for the refactoring! 👍 

Just to be fair regarding the shebang line again, even though the above link mentions some possible issues, most npm module binaries include such a shebang line, as you [noted](https://docs.npmjs.com/files/package.json#bin). Therefore it is probably a very corner case…